Project.toml

```diff
@@ -1,18 +1,18 @@
 name = "BitIntegers"
 uuid = "c3b6d118-76ef-56ca-8cc7-ebb389d030a1"
 authors = ["Rafael Fourquet <fourquet.rafael@gmail.com>"]
-version = "0.2.3"
+version = "0.2.4"
+
+[deps]
```

Comment:
Just a small thing, but could you minimize the change to Project.toml? I makes it clearer that only the version changed (I often have Pkg reorder the lines in this file and I find it rather annoying... maybe it was the case here?)

Reply:
I can drop that commit. Is there any particular reason why you don't want to use Pkgs canonical ordering/formatting?

Reply:
I just tend to be annoyed by Pkg changing my ordering, and didn't really consider that Pkg has a "canonical" ordering...  so I might just have been reacting to that ;-)

Reply:
Okay. Thanks for explaining!